### ISSUE TITLE
Relieved by nerve block

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#9867]
**Preferred term label:**

**Synonyms**
Positive response to anesthetic block of the nerve

**Definition (free text, please give PubMed ID)**
Positive response to anesthetic block of the nerve

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pudendal neuralgia ORPHA:60039

**Your nano-attribution (ORCID)**
PMID 24716710 : The Nantes criteria 20 are recommended for the diagnosis 
of pudendal neuropathy. There are fi ve essential diagnostic 
criteria: 
(1) Pain is expressed in the anatomical territory of the 
pudendal nerve; 
(2) Pain is aggravated by sitting; 
(3) The patient does not wake up during the night due to the 
absence of nocturnal pain; 
(4) There is no objective sensory loss on clinical examination; 
and 
(5) There is positive response to **anesthetic block of the 
pudendal nerve.**

PMID 17828787 : The five essentials diagnostic criteria are: (1) Pain in the anatomical territory of
the pudendal nerve. (2) Worsened by sitting. (3) The patient is not woken at night by the pain. (4) No objective
sensory loss on clinical examination. (5) **Positive anesthetic pudendal nerve block**

PMID 22110274 : (5) Pain should be relieved by anesthetic infiltration of the pudendal nerve. 
This is an essential criterion, but it lacks specificity as pain related to any perineal disease may be **relieved by 
pudendal nerve block**

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000068
- label: Ameliorated by nerve block
- parents: [Ameliorated by (HP:0025254)](https://hpo.jax.org/app/browse/term/HP:0025254)
- pmid: 
- def: Refers to a sign or symptom that is improved or made more bearable by anesthetic block of a nerve.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9867
- synonyms:
